packages/react-components/src/atoms/LinkConditional.tsx

```diff
@@ -0,0 +1,10 @@
+import { ComponentProps } from 'react';
+
+import Link from './Link';
+
+const ConditionalLink: React.FC<ComponentProps<typeof Link>> = ({
+  children,
+  ...props
+}) => (props.href ? <Link {...props}>{children}</Link> : <>{children}</>);
+
+export default ConditionalLink;
```

Comment:
there is a mismatch between the filename and the component name.
but I didn't find where are we using this and why its necessary

Reply:
I need it in packages/react-components/src/templates/TeamProfileOutputs.tsx as we may not always have emails to create a mailto with

Reply:
I thought both components would need the link for the pm emails link but we decided against the extra query on users. I could potentially move this into TeamProfileOutputs.tsx